ordered_associative_task/student/src/impl.cpp

```diff
@@ -1 +1,6 @@
 #include "impl.h"
+std::map<size_t,size_t> mapConverter={{29,1},{49,2},{69,3},{89,4},{SIZE_MAX,5}};
```

Comment:
Where SIZE_MAX is defined?

Reply:
Is not SIZE_MAX the maximum value of size_t?
https://en.cppreference.com/w/c/types/limits